src/components/grid/grid.test.js

```diff
@@ -31,267 +32,167 @@ const viewportSize = (sizeOfViewport) => {
-          <pod alignTitle="left" border padding="medium" variant="primary">
-            1
-          </pod>
-        </GridItem>
-        <GridItem
-          alignSelf="stretch"
-          justifySelf="stretch"
-          gridColumn="1 / 6"
-          gridRow="2 / 3"
-        >
-          <pod alignTitle="left" border padding="medium" variant="primary">
-            2
-          </pod>
-        </GridItem>
-        <GridItem
-          alignSelf="stretch"
-          justifySelf="stretch"
-          gridColumn="7 / 13"
-          gridRow="4 / 5"
-        >
-          <pod alignTitle="left" border padding="medium" variant="primary">
-            3
-          </pod>
-        </GridItem>
-      </GridContainer>
-    </>
+    <GridContainer>
+      <GridItem alignSelf="stretch" justifySelf="stretch">
+        <Pod>Item 1</Pod>
+      </GridItem>
+      <GridItem
+        alignSelf="stretch"
+        justifySelf="stretch"
+        gridColumn="1 / 6"
+        gridRow="2 / 3"
+      >
+        <Pod>Item 2</Pod>
+      </GridItem>
+      <GridItem
+        alignSelf="stretch"
+        justifySelf="stretch"
+        gridColumn="7 / 13"
+        gridRow="4 / 5"
+      >
+        <Pod>Item 3</Pod>
+      </GridItem>
+    </GridContainer>
   );
 };
 
 context("Testing Grid component", () => {
   describe("should render Grid component", () => {
     it.each([
-      [1, "1 /1", "1 / 13", 1862],
+      [1, "auto / auto", "1 / 13", 1862],
       [2, "2 / 3", "1 / 6", 752],
       [3, "4 / 5", "7 / 13", 911],
     ])(
-      "should render Grid viewport to default and verify pod width and height",
-      (index, row, col, podWidth) => {
-        CypressMountWithProviders(<GridComponentSized />);
+      "when viewport size is default, grid item %s should have correct grid-row, grid-column and width",
+      (itemNumber, row, col, expectedWidth) => {
+        CypressMountWithProviders(<GridLayoutExample />);
 
         viewportSize("default");
-        if (index === 1) {
-          gridPod(index).should("have.css", "grid-row", "auto / auto");
-          gridPod(index).should("have.css", "grid-column", "auto / auto");
-          gridPod(index)
-            .invoke("css", "width")
-            .then(parseFloat)
-            .then(($el) => {
-              expect($el).to.be.gte(podWidth);
-              expect($el).to.be.lt(podWidth + 1);
-            });
-        } else {
-          gridPod(index).should("have.css", "grid-row", row);
-          gridPod(index).should("have.css", "grid-column", col);
-          gridPod(index)
-            .invoke("css", "width")
-            .then(parseFloat)
-            .then(($el) => {
-              expect($el).to.be.gte(podWidth);
-              expect($el).to.be.lt(podWidth + 1);
-            });
-        }
+        gridItem(itemNumber - 1).should("have.css", "grid-row", row);
+        gridItem(itemNumber - 1).should("have.css", "grid-column", col);
+        gridItem(itemNumber - 1)
+          .invoke("css", "width")
```

Comment:
I would rather not use `invoke` here, just use the `jQuery` method using 

`gridItem(itemNumber - 1).then(($el) => {
     expect($el.css("width"))...
})`